UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs

```diff
@@ -96,21 +125,175 @@ public void SaveGLTFandBin(string path, string fileName)
 			gltfFile.Close();
 			binFile.Close();
 #endif
-
-			foreach (var image in _images)
-			{
-				Debug.Log(image.name);
-				var renderTexture = RenderTexture.GetTemporary(image.width, image.height);
-				Graphics.Blit(image, renderTexture);
-				RenderTexture.active = renderTexture;
-				var exportTexture = new Texture2D(image.width, image.height);
-				exportTexture.ReadPixels(new Rect(0, 0, renderTexture.width, renderTexture.height), 0, 0);
-				exportTexture.Apply();
-				File.WriteAllBytes(Path.Combine(path, image.name + ".png"), exportTexture.EncodeToPNG());
-			}
+            exportImages(path);
```

Comment:
ExportImages is more the C# style and what we have been using in this code base

Reply:
Fixed